Mathlib/MeasureTheory/Measure/Lebesgue/Complex.lean

```diff
@@ -3,9 +3,8 @@ Copyright (c) 2021 Yury Kudryashov. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yury Kudryashov
 -/
-import Mathlib.Analysis.SpecialFunctions.Integrals
+import Mathlib.Analysis.SpecialFunctions.PolarCoord
```

Comment:
When you import this, you import a lot of nontrivial things. Could you instead leave the file as it was, and put your results in the file `Mathlib.Analysis.SpecialFunctions.PolarCoord`?

Reply:
I moved everything to `Mathlib.Analysis.SpecialFunctions.PolarCoord` as you suggested. That means of course that I need to import `Mathlib.MeasureTheory.Measure.Lebesgue.Complex` in this file but I guess that's okay.